### ISSUE TITLE
ability to change the appointment start and finish times on the fly.

### ISSUE THREAD

**Issue by sshelagh** [russells/makediary#3]
Some people, like tradesmen, start early in the morning eg 7:30 am. How hard would it be to change the appointment times through a command-line switch? I changed the source to achieve what I was looking for, but it would be good to have it as an option.
![diarypage](https://cloud.githubusercontent.com/assets/168074/11465480/135f77a0-978d-11e5-9419-837fb31ee12e.png)

**Comment by russells** [comment]
Good idea.  I'll look into it.  I gather you don't need it for this year.

**Comment by sshelagh** [comment]
No, but I went in and changed the personalinfo.py (ish) file to suit my
husband's day better. I also put a Notes at the bottom. Looks goo.

I thought I would have a go at making a function in python that would
created the list based on start time finish time and incrementt just incase
someone would like 1/2 hour appointments. If you do that it would be great
but I will still try to work that out as an exercise.

But no. I don't need it for this year as I have already printed out the
diary.

Cheers

Shelagh

**Comment by sshelagh** [comment]
An example for you.
